src/test/java/edu/harvard/iq/dataverse/engine/command/impl/RestrictFileCommandTest.java

```diff
@@ -131,6 +133,99 @@ public void testRestrictRestrictedFile() throws Exception {
         
     }
     
+    @Test
```

Comment:
Basically mirror images of the existing tests - just verified that the logic is symmetric.